src/meta/inode.rs

```diff
@@ -0,0 +1,66 @@
+use crate::color::{ColoredString, Colors, Elem};
+use std::fs::Metadata;
+
+#[derive(Debug, PartialEq, Eq, Copy, Clone)]
+pub struct INode {
+    index: u64,
+    valide: bool,
```

Comment:
I'm guessing this is supposed to be `valid`.

In any case, wouldn't it be better to just use `Option` for `index` than have a separate flag?

Reply:
updated to 

```rust
pub struct INode {
    index: Option<u64>,
}
```